pom.xml

```diff
@@ -1099,6 +1099,12 @@ limitations under the License.
                   <dataFileIncludes>
                     <dataFileInclude>**/jacoco.exec</dataFileInclude>
                   </dataFileIncludes>
+                  <excludes>
+                    <exclude>org.projectnessie.versioned.tests.AbstractITVersionStore</exclude>
+                    <exclude>org.projectnessie.versioned.impl.AbstractITTieredVersionStore</exclude>
+                    <exclude>org.projectnessie.versioned.impl.AbstractTestStore</exclude>
+                    <exclude>org.projectnessie.versioned.impl.AbstractTieredStoreFixture</exclude>
```

Comment:
Should `org.projectnessie.versioned.impl.SampleEntities` be added as well (b/c this is basically a list of all classes in versioned-tests and versioned-tiered-tests)?

Reply:
Since they are not actual test classes there is sensible code coberage for those classes (bumps our coverage ;-)